### PR TITLE
Revert "Wrap examples in shortcode"

### PR DESCRIPTION
**Pull request by stack72**
Reverts pulumi/pulumi-terraform-bridge#130

### CODE REVIEW

File: pkg/tfgen/docs.go [hunk 1]
```diff
@@ -425,20 +418,9 @@ func parseTFMarkdown(g *generator, info tfbridge.ResourceOrDataSourceInfo, kind
...
 					}
 				}
-
-				description := strings.Join(subsection, "\n") + "\n"
-				if headerIsExampleUsage {
-					// Wrap each example in shortcode.
-					description = "{{% example %}}\n" + description + "{{% /example %}}\n"
-				}
-				ret.Description += description
+				ret.Description += strings.Join(subsection, "\n") + "\n"
```
Comment (praneetloke) [inline pkg/tfgen/docs.go]: Is this change intentional?

Comment (stack72) [reply pkg/tfgen/docs.go]: @praneetloke i just hit revert on the original PR - this wasnt manually reverted

Comment (stack72) [reply pkg/tfgen/docs.go]: Looks like it was done in the original PR https://github.com/pulumi/pulumi-terraform-bridge/pull/130/files#diff-9cad96fc5e12fbb60a4c1204a57f5035L421

Comment (stack72) [reply pkg/tfgen/docs.go]: this was just a GH based revert - not manual

Comment (praneetloke) [reply pkg/tfgen/docs.go]: Ah yes I see (it was moved a line below where it originally was.) Thanks for confirming, Paul!
